label_maker/utils.py

```diff
@@ -42,13 +43,25 @@ def get_tile_tif(tile, imagery, dest_folder):
     http://www.cogeo.org/in-depth.html
     """
     bound = bounds(*[int(t) for t in tile.split('-')])
+    imagery_offset = imagery_offset or [0, 0]
     with rasterio.open(imagery) as src:
         x_res, y_res = src.transform[0], src.transform[4]
-        proj_to = Proj(**src.crs)
+        p1 = Proj({'init': 'epsg:4326'})
+        p2 = Proj(**src.crs)
+
+        # offset our imagery in the "destination pixel" space
+        offset_bound = dict()
+        deg_per_pix_x = (bound.west - bound.east) / 256
+        deg_per_pix_y = (bound.north - bound.south) / 256
```

Comment:
Should you add a period after `256`? Would these ever need to be floats in python2?

Reply:
We don't support python 2

Reply:
Nice, good choice us